lessons/python_basics/lesson.md

```diff
@@ -0,0 +1,57 @@
+# (Very, Very Brief) Introduction to Python
```

Comment:
Just out of curiosity, but are you planning on going over the entire `practice_code.py` in the code-along? If you are, I am a little confused by the title "very, very brief". You have a lot of material here... To put it in perspective, Software Carpentry workshops go over a similar amount of code/topics **in 4 hours**. In teaching that I've done for introductions to languages/concepts, I get through *maybe* 60-100 lines of code in *2 hours*. It's hard to predict what questions will come up and what confusions there will be during the lesson.

I definitely don't want to discourage you, but there is a disconnect between the title and the content. Maybe change the title to just "Introduction to Python"? Just my 2 cents.